cli/install/viz.go

```diff
@@ -0,0 +1,67 @@
+package install
+
+// Viz defines the primary Conduit Grafana dashboard, installed via the `conduit install` command.
+const Viz = `{
+      "rows": [
+        {
+          "collapse": false,
+          "height": "50px",
+          "panels": [
+            {
+              "content": "<div>\n  <div style=\"position: absolute; top: 0, left: 0\">\n    <a href=\"https://conduit.io\" target=\"_blank\"><img src=\"https://conduit.io/images/conduit-primary-white.svg\" style=\"height: 30px;\"></a>\n  </div>\n  <div style=\"position: absolute; top: 0; right: 0; font-size: 15px\">\n    <a href=\"https://conduit.io\" target=\"_blank\">Conduit</a> is a next-generation ultralight service mesh for Kubernetes.\n    <br>\n    Need help? Join us on <a href=\"http://slack.linkerd.io\" target=\"_blank\">Slack</a>.\n  </div>\n</div>",
+              "height": "1px",
+              "id": 14,
+              "links": [],
+              "mode": "html",
+              "span": 12,
+              "title": "",
+              "transparent": true,
+              "type": "text"
+            }
+          ],
+          "showTitle": false
+        },
+        {
+          "collapse": false,
+          "height": 250,
+          "panels": [
+            {
+              "legend": {
+              },
+              "lines": true,
+              "targets": [
+                {
+                  "expr": "sum(irate(responses_total[20s])) by (target_deployment)",
+                  "format": "time_series",
+                  "intervalFactor": 1,
+                  "legendFormat": "{{target_deployment}}",
```

Comment:
Note that Grafana uses `{{` to identify dashboard template variables, which disallows us from in-linining Grafana dashboard JSON files directly in Go templates. this is probably a fine as it's best to keep these files separate.

If we ever end up needing to templatize these JSON files, we have two options:
1) escape them: `{{"{{"}}`
2) modify our template parsing to use non-default delimiters: https://golang.org/pkg/html/template/#Template.Delims